### ISSUE TITLE
Error with date format

### ISSUE THREAD

**Issue by brunj7** [jirikadlec2/waterml#13]
Hi,

I am having an error with date conversion when trying to retrieve data from SCAN sites

code example:

```R
library(tidyverse)
library(stringr)
library(WaterML)

#get the list of supported CUAHSI HIS services
services <- GetServices()

# Get a specific station
server <- services %>% 
  # Get all the SCAN network
  filter(str_detect(.$networkName, "SCAN")) %>%
  select(url) %>% pull()

# get all variables
variables <- GetVariables(server)

# get all sites
sites <- GetSites(server)

# get full site info for all sites using the GetSiteInfo method
siteinfo <- GetSiteInfo(server, "SCAN:2069_NH_SCAN")

# Do a specific query
Temp <- GetValues(server, siteCode="SCAN:2069_NH_SCAN",
                  variableCode="SCAN:TOBS_D")
```

error message: 

```R
[1] "downloading values from: http://hydroportal.cuahsi.org/Scan/cuahsi_1_1.asmx ..."
[1] "download time: 3.936 seconds, status: Success"
[1] "reading data values WaterML ..."
Error in as.POSIXct.default(DateTimeUTC, format = "%Y-%m-%dT%H:%M:%S",  : 
  do not know how to convert 'DateTimeUTC' to class “POSIXct”
```

**Comment by jirikadlec2** [comment]
Thank you for reporting the issue.
I tried running the code example and I am getting the same issue.
I investigated the WaterML data (http://hydroportal.cuahsi.org/SCAN/cuahsi_1_1.asmx/GetValuesObject?location=SCAN:2069_NH_SCAN&variable=SCAN:TOBS_D&startDate=&endDate=&authToken=) and the error is related to the dateTimeUTC attribute not present in the data.
I'll try to submit a fix asap.

**Comment by jirikadlec2** [comment]
Hi,
The error has been fixed in WaterML R package version 1.9.0 
, commit https://github.com/jirikadlec2/waterml/commit/d0f0489fad88045d66c8dfc4c6d0f5a893920161

The code example now returns valid results.
Apologies it took so long to fix the error.

Best regards,
Jiří
